### PR TITLE
Remove skeptic

### PR DESCRIPTION
**Pull request by lu-zero**
It is unmaintained and fails to build tests.

Same as https://github.com/vincent-herlemont/native_db/pull/286.

### CODE REVIEW

**Comment by vincent-herlemont** [comment]
@lu-zero Thank you for your PR!

An issue has been opened for this (https://github.com/vincent-herlemont/native_model/issues/76). It is important that the code in the Readme can continue to be tested by the CI. Therefore, in the same PR for removal, it is necessary to find a way to replace it.

In the case of https://github.com/vincent-herlemont/native_db/pull/286 it is not the same thing because `#[doc = include_str!("../../README.md")]` was already done which allowed testing the code in the Readme.

File: README.md [hunk 1]
```diff
@@ -170,7 +242,7 @@ native_model = { version = "0.4", features = [ "rmp_serde_1_3" ] }
 
 ```rust
 #[derive(Clone, Default, serde::Deserialize, serde::Serialize)]
-#[native_model(id = 1, version = 1, with = native_model::rmp_serde_1_3::RmpSerde)]
+#[native_model::native_model(id = 1, version = 1, with = native_model::rmp_serde_1_3::RmpSerde)]
```
Comment (vincent-herlemont) [inline README.md:245]: To be consistent with the other examples, is it possible to import the library `use native_model::native_model;` to have?
```suggestion
#[native_model(id = 1, version = 1, with = native_model::rmp_serde_1_3::RmpSerde)]
```

Comment (lu-zero) [reply README.md:245]: Sure, let me quickly update it.

Comment (vincent-herlemont) [reply README.md]: Perfect :)

**Pull request merged**